t/lib/test_admin.lua

```diff
@@ -108,14 +108,15 @@ function _M.comp_tab(left_tab, right_tab)
     local err
     dir_names = {}
 
+    local _
     if type(left_tab) == "string" then
-        left_tab, err = json.decode(left_tab)
+        left_tab, _, err = json.decode(left_tab)
```

Comment:
where to get this API manual?

the new way seems wrong. I think the correct style should be `local res, err = json.decode(str)`

Reply:
@membphis 
For dkjson, the second argument is `pos`, not `err`.

Reply:
got it, many thanks for your explain